tests/integration.rs

```diff
@@ -234,3 +236,50 @@ fn read_from_stdin() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn read_from_directory() -> Result<(), Box<dyn std::error::Error>> {
+    let target = PathBuf::from("target/integration");
+    let _ = std::fs::create_dir(&target);
```

Comment:
This creates test directories in the current working dir, without checking first whether they already exist. I got bitten by that in #61. For extra robustness, can you do that in a temporary directory (see [tempfile::tempdir()](https://docs.rs/tempfile/3.10.1/tempfile/fn.tempdir.html)) to ensure pristine test data, and that it gets deleted at the end of the test?